auth/auth.go

```diff
@@ -25,7 +54,10 @@ func Auth(rw http.ResponseWriter, req *http.Request) bool {
 		return false
 	}
 
-	if token.Claims["hostUuid"] != config.Config.HostUuid {
+	if token.Claims["reportedUuid"] != nil && token.Claims["reportedUuid"] != config.Config.HostUuid {
```

Comment:
This was because of the corresponding Cattle change. Removed this change to have a check only on HostUuid.